pyartcd/pyartcd/pipelines/gen_assembly.py

```diff
@@ -212,8 +225,8 @@ async def _create_or_update_pull_request(self, assembly_definition: OrderedDict)
 @cli.command("gen-assembly")
 @click.option("--data-path", metavar='BUILD_DATA', default=None,
               help=f"Git repo or directory containing groups metadata e.g. {constants.OCP_BUILD_DATA_URL}")
-@click.option("-g", "--group", metavar='NAME', required=True,
-              help="The group of components on which to operate. e.g. openshift-4.9")
+@click.option("--version", metavar='NAME', required=True,
```

Comment:
I would like to keep `--group` option since it is more generic than `--version`.

Reply:
Keeping group back

Reply:
How is `openshift-4.y` more generic than a bare version like `4.y`? If you pass in the version, you can build the group as `f'openshift-{version}'`, but if you pass in the group you'll have to `group.split('-')[1]` which looks tremendously ugly to me.